doc/source/license.rst

```diff
@@ -0,0 +1,39 @@
+:tocdepth: -1
```

Comment:
THis file needs to be changed by copy-pasting the license text from the other license file in here to replace the license text here.